### PR TITLE
Base of location Based Suspicion

### PR DESCRIPTION
**Pull request by KyriakosNU**

### CODE REVIEW

File: CSC8503/SuspicionSystem/SuspicionMetre.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+	class SuspicionMetre
+	{
+	public:
+		enum SusBreakpoint
+		{
+			low, mid, high
+		};
+
+		SusBreakpoint getSusBreakpoint(float inSusMetre);
```
Comment (Nijelous) [inline CSC8503/SuspicionSystem/SuspicionMetre.h:14]: Capitals for Functions

File: CSC8503/SuspicionSystem/SuspicionMetre.h [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	private:
+		std::map<float, SusBreakpoint> SusBreakpointMap = { { 66 , high} ,
```
Comment (Nijelous) [inline CSC8503/SuspicionSystem/SuspicionMetre.h:16]: mSusBreakpointMap

**Review by Nijelous — CHANGES_REQUESTED** [review]
A couple quick changes but the code looks solid, and I assume it works.

**Review by Nijelous — APPROVED** [review]

**Review by fm19121 — APPROVED** [review]
curly brackets

**Pull request merged**